packages/styled-docs/pages/formcontrol.mdx

```diff
@@ -0,0 +1,15 @@
+# FormControl
```

Comment:
Is there any FormControl component that we build?

Reply:
This file should be in another PR. Thank you for your notice.

Reply:
Removed in 1b44e38be4c986fcfa6d54acb29a6467d3fea971